runtime/parachains/src/hrmp.rs

```diff
@@ -412,6 +412,25 @@ decl_module! {
 	}
 }
 
+use xcm_executor::traits::ExecuteHrmp;
+use xcm::v0::{Result as XcmResult};
+impl<T: Config> ExecuteHrmp for Module<T> {
+	fn hrmp_init_open_channel(sender: u32, recipient: u32, max_message_size: u32, max_capacity: u32) -> XcmResult {
+		Self::init_open_channel(sender.into(), recipient.into(), max_capacity, max_message_size).map_err(|_| ().into())
```

Comment:
```suggestion
		Self::init_open_channel(sender.into(), recipient.into(), max_capacity, max_message_size).map_err(|_| ().into())
```

Right now, this is returning the XcmError::Undefined if a runtime error is returned. What error should it use?